src/validations/profile.ts

```diff
@@ -59,8 +66,59 @@ export const wearableUrns: Validation = validationAfterADR75({
   }
 })
 
+export const profileEmotes: Validation = validationAfterADR74({
+  validate: async (components, deployment) => {
+    const allAvatars = deployment.entity.metadata?.avatars ?? []
+    for (const avatar of allAvatars) {
+      const allEmotes = avatar.avatar.emotes ?? []
+      for (const { slot, urn } of allEmotes) {
+        const parsed = await parseUrn(urn)
+        if (!parsed)
+          return validationFailed(
+            `Each profile emote pointer should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (${urn})`
+          )
+        if (slot < 0 || slot > 9) {
```

Comment:
should we validate that `slot` is a number too?

Reply:
That validation would be part of the `metadata schema` validation.